pandas/core/algorithms.py

```diff
@@ -1159,7 +1159,7 @@ def compute(self, method):
         n = min(n, narr)
 
         kth_val = algos.kth_smallest(arr.copy(), n - 1)
-        ns, = np.nonzero(arr <= kth_val)
+        (ns,) = np.nonzero(arr <= kth_val)
```

Comment:
I guess this is from black. Can you use the version in `environment.yml` (`conda install black<=19.3b0`), so no unrelated changes are applied.